common/json_schema.py

```diff
@@ -1,14 +1,35 @@
 from marshmallow import Schema,fields
 
-class Faq_Schema(Schema):
-    """
+"""
     schema.dump = used for converting the automap.faqs object to a dictionary good for returning ie cleaning unncessary fields
-    schema.validate(request.get_json(force=True)) = used to validate if all the data required for updating and creating the faq is present.
-    """
+    schema.validate(request.get_json(force=True)) = used to validate if all the data required for updating and creating the faq is present.4
```

Comment:
Are these actually used in any of the files? I'm not seeing them in this file at all?

Reply:
They are used to validate the data given in the request. Check PUT or POST request handlers for any endpoint.